2_feature_importance/2_feature_importance.ipynb

```diff
@@ -0,0 +1,500 @@
+{
```

Comment:
_Commented on notebook `2_feature_importance/2_feature_importance.ipynb` Cell 24 From line 1 to 2_
```python
shap.plots.beeswarm(shap_values)
shap.plots.bar(shap_values)
```
It would be nice to discuss what these plots are producing below. To build a narrative, I think it would be best to start with a waterfall plot of one prediction, where you can explain a bit more what are those SHAP values (i.e. they show how much the variable values affect the deviation from the average to the true output). Then go to one of those two plots you have in here, where all individual predictions are aggregated (in different ways), and leave the remaining one for the exercise.

_Respond and view the context [here](https://gitnotebooks.com/oriol-pomarol/codegeo_workshops/pull/1)._

Reply:
This is part of the presentation